history-of-quakers/modernized/28-appendix.adoc

```diff
@@ -729,10 +728,10 @@ he was slow in taking off that where the gold was, and the marshal thinking it t
 and thinking there had been no more, bid him put them on again.
 And upon a certain time he gave it me in half a little white loaf,
 when we stood with the soldiers waiting to be called.
-And they seeing the book called '`The way to the kingdom,`' that it was printed at Amsterdam,
+And they seeing the book called [.book-title]#The Way to the Kingdom,# that it was printed at Amsterdam,
 asked me, '`What books I brought from there to Paltz, and how many?`'
 And one morning having sent for one of the priests.
-He got that written book of George Fox`'s of the Apostasy of Christendom,
+He got that written book of George Fox`'s of the [.book-title]#Apostasy of Christendom,#
```

Comment:
I wasn't sure if this was the actual title of the book...I tried to look it up, but couldn't find anything and this appendix isn't in the orginal PDF that I was using.